### ISSUE TITLE
Addressing data security concerns with datapackagist

### ISSUE THREAD

**Issue by Stephen-Gates** [frictionlessdata/datapackage-ui#200]
As a publisher of open data I need to get approval to publish my organisations open data. Part of this process includes getting approval for the metadata that accompanies the data.

As a user of datapackagist, I need to send my unreleased data across the internet to help generate the metadata.

I want to avoid complicating my data release process into a multi-step process e.g. 
- approval to release data to datapackagist,
- separate later approval of metadata created using datapackagist
- publish data and metadata to an open data portal.

Possible solutions: 
- Provide a statement on Datapackagist about how/if the data is stored, secured and disposed of.
- Provide a packaged version of the software that can run locally, thus avoiding the need to send the data over the internet.
- Do nothing and let the data publisher adjust their process. 

Notes: 
- I had similar issues raised at work when exploring tools like [CSVLint](http://csvlint.io) (where [it showed the results of CSV validations](http://csvlint.io/validation)) and [Good Tables](http://goodtables.okfnlabs.org). 
- Given this concern, Datapackagist (and CSV validation tools) may be used more to create data packages and validate open data that is already published.

**Comment by pwalsh** [comment]
Hey @Stephen-Gates this is an excellent topic. I'm curious about this issue more widely. DataPackagist has been designed around an initial use case by regional governments in the UK, and GoodTables is actually in use as part of similar publishing flows in some departments of the UK Govt.

For DataPackagist, it may be quite simple to package it as a desktop app using [Electron](http://electron.atom.io/), which is something I've been considering for use for these tools over the last months.

@richhall @rgrp any thoughts / comments?

**Comment by Stephen-Gates** [comment]
As the trend in publishing open data moves from quantity to quality, many groups I know are looking to provide better metadata and to test the quality of their data before release. Given the above concern, some groups are turning to internal tools (e.g. FME) to do this work. This has issues such as:

- cost of tools
- complex to learn
- waiting on stretched IT staff to assist.

Creating an offline tool that combines Datapackagist and GoodTables to help publishers release quality metadata and data that complies with the schema would really help.

**Comment by pwalsh** [comment]
Thanks @Stephen-Gates I'm really very interested in this idea, and our tooling around data packages and csv validation is getting a lot of focus over the last months, which will continue throughout 2016.

**Comment by Stephen-Gates** [comment]
Now working on Electron app. Can close.

**Comment by pwalsh** [comment]
Closing in favour of https://github.com/frictionlessdata/datapackage-ui/issues/227
